packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -1050,6 +1050,13 @@ describe("<DateRangePicker>", () => {
             render({ timePrecision: "minute", defaultValue: defaultRange }).clickShortcut();
             assert.isTrue(DateUtils.areSameTime(onChangeSpy.firstCall.args[0][0] as Date, defaultRange[0]));
         });
+
+        it("selecting and unselecting a day doesn't change time", () => {
+            const leftDatePicker = render({ timePrecision: "minute", defaultValue: defaultRange }).left;
+            leftDatePicker.clickDay(5);
+            leftDatePicker.clickDay(5);
+            assert.isTrue(DateUtils.areSameTime(onChangeSpy.secondCall.args[0][0] as Date, defaultRange[0]));
```

Comment:
this is not what i expected... it's about selecting, clearing, then re-selecting should remember the time. please revise.

Reply:
Hey Gilad, the defaultValue is already specified (which is on March 5th), so the the 5th day is already selected. Clicking twice should then clear, and re-select. Or are you suggesting to not use any default values, and have it: select day, select time, unselect day, re-select day?

Reply:
oh gotcha, didn't notice that 👍